datasets/polsum/README.md

```diff
@@ -0,0 +1,172 @@
+---
+annotations_creators: 
+- expert-generated
+language_creators: 
+- expert-generated
+languages: 
+- pl
+licenses: 
+- cc-by-3.0
+multilinguality: 
+- monolingual
+size_categories: 
+- n<1K
+source_datasets: 
+- original
+task_categories: 
+- conditional-text-generation
+task_ids: 
+- summarization
+---
+
+# Dataset Card for polsum
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** http://zil.ipipan.waw.pl/PolishSummariesCorpus
+- **Repository:** http://zil.ipipan.waw.pl/PolishSummariesCorpus
+- **Paper:** http://nlp.ipipan.waw.pl/Bib/ogro:kop:14:lrec.pdf
+- **Leaderboard:** [Needs More Information]
+- **Point of Contact:** [Mateusz Kopeć](http://zil.ipipan.waw.pl/MateuszKopec)
+
+### Dataset Summary
+
+The Corpus contains a large number of manual summaries of news articles,
+with many independently created summaries for a single text. Such approach is supposed to overcome the annotator bias, which is often described as a problem during the evaluation of the summarization algorithms against a single gold standard.
+
+### Supported Tasks and Leaderboards
+
+[Needs More Information]
+
+### Languages
+
+Polish
+
+## Dataset Structure
+
+### Data Instances
+
+[Needs More Information]
```

Comment:
can you add one example from the dataset here ?

You can get one using `load_dataset` and taking the first sample `dataset["train"][0]` for example